component_information/actuators.schema.json

```diff
@@ -17,17 +17,17 @@
             "properties": {
                 "min": {
                     "type": "number",
-                    "minium": -1,
+                    "minimum": -1,
```

Comment:
This is currently only used by PX4. Was the change made upstream too?

Reply:
https://github.com/ArduPilot/mavlink/blob/master/component_information/actuators.schema.json#L20
https://github.com/mavlink/mavlink/blob/master/component_metadata/actuators.schema.json#L20

The directory is renamed `component_metadata` --> `component_information` and the spelling is correct on upstream and incorrect on downstream.